lib/forms/submission_statuses/formatter.rb

```diff
@@ -46,7 +48,9 @@ def build_submissions_map(submissions)
             message: nil,
             status: nil,
             created_at: submission.created_at,
-            updated_at: nil
+            updated_at: nil,
+            pdf_support: PdfUrls.new(form_id: submission.form_type,
```

Comment:
Just a suggestion.  Update this...
```ruby
  pdf_support: PdfUrls.new(form_id: submission.form_type,
                           submission_guid: submission.benefits_intake_uuid).supported?
```

To someting like this...
```ruby
pdf_support: pdf_supported?(submission)
```
```ruby
def pdf_supported?(submission)
  PdfUrls.new(
    form_id: submission.form_type,
    submission_guid: submission.benefits_intake_uuid
  ).supported?
end
```

Reply:
Also, I'm not sure the class name `Formatter` is the right choice (I originally named it), but we can rename in the future if necessary.

Reply:
agree... can't think of a better name atm 🤔 .  It's more like a "Submission Status Collection Builder"? I'll leave as-is for now 👍